python-sdk/examples/fetch_policies_demo.py

```
@@ -0,0 +1,185 @@
+import subprocess
+
+from dotenv import load_dotenv
+import langwatch
+from langwatch import FetchPolicy
+
+
+def run_cli(command: list, cwd: Path):
+    """Run CLI command and handle errors."""
+    print(f"▶️  {' '.join(command)}")
+    try:
+        result = subprocess.run(
```

Ruff (0.14.8)

`subprocess` call: check for execution of untrusted input

(S603)